covid-19/total-dod/js/spline.js

```diff
@@ -86,12 +86,10 @@ Highcharts.chart("hcContainer", {
       overflow: 'none',
       formatter: function () {
         console.log(this.series.chart.series)
+        
         let s = this.series.chart.series
-        let total = s[3].points[s[3].points.length - 1].y
-        // s.forEach((serie, i) => {
-        //   console.log(serie)
-        //   total += serie.points[serie.points.length - 1].y
-        // })
+        let total = s[2].points[s[2].points.length - 1].y
```

Comment:
This is the wrong series.  They would like it to show cumulative cases, which is `s[0]`